recipes/pot/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set name = "POT" %}
+{% set version = "0.3.1" %}
+{% set sha256 = "1c226e359e81ca08e593012b9022d64041318d0bd7aafcb30185f1c4f02304af" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - toolchain
+    - cython
+    - numpy x.x
+    - scipy
```

Comment:
Aren't `setup.py`'s `install_requires` generally needed in the build step, because otherwise `setup.py install` will try to download them?